src/dmd/dsymbolsem.d

```diff
@@ -1148,8 +1148,15 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             {
                 AliasDeclaration ad = imp.aliasdecls[i];
                 //printf("\tImport %s alias %s = %s, scope = %p\n", toPrettyChars(), aliases[i].toChars(), names[i].toChars(), ad._scope);
-                if (imp.mod.search(imp.loc, imp.names[i]))
+                Dsymbol importedSymbol = imp.mod.search(imp.loc, imp.names[i]);
+                if (importedSymbol)
                 {
+                    // Deprecated in 2018-01.
+                    // Change to error in 2019-01.
+                    // @@@DEPRECATED_2019-01@@@.
+                    import dmd.access : symbolIsVisible;
+                    if (!symbolIsVisible(sc, importedSymbol))
+                         imp.mod.deprecation(imp.loc, "member '%s' is not accessible", imp.names[i].toChars());
```

Comment:
Sorry, but I found one more thing.  We need to put tick marks around the %s.  (e.g. \`%s\`).  Remove the single quotes.

Reply:
And don't forget to update the error messages in the test case too.

Reply:
Oh, missed that one too. Thanks!